src/modules/Dashboard/features/RespondentData/RespondentDataReview/Review/Review.types.ts

```diff
@@ -1,11 +1,14 @@
-import { DecryptedActivityData, EncryptedAnswerSharedProps } from 'shared/types';
+import { DecryptedActivityData } from 'shared/types';
+import { EncryptedActivityAnswer } from 'modules/Dashboard/api';
 
 import { Answer } from '../RespondentDataReview.types';
 
 export type ReviewProps = {
   isLoading: boolean;
   selectedAnswer: Answer | null;
-  activityItemAnswers: DecryptedActivityData<EncryptedAnswerSharedProps>['decryptedAnswers'] | null;
+  activityItemAnswers:
```

Comment:
let's create a separate type for this and use it in RespondentDataReview.tsx#L76 as well?